src/utils/fullScreenWidthPixels.js

```diff
@@ -1,5 +1,9 @@
 import devicePixelRatio from "./devicePixelRatio";
 
+/**
+ * This value used is based on (and thus optimized for) the Lighthouse mobile emulation.
```

Comment:
FYI: "This value used is based on" seems off. Maybe "This value is based on"? Or maybe shorten it to `Based on (and thus optimized for) the Lighthouse mobile emulation.`

Reply:
True. Will force push.